### DISCUSSION TITLE
ERGO Node configuration

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1176] [Q&A]
Hello folks, 
hope someone can lighten me up. How do I have to configure the ERGO full node when I have a dedicated payout? 
There is an option "miningPubKeyHex" for the ERGO node itself. Is this option needed for the ERGO nodes or obsolet, as it is set from the address in the configuration and via API? 

Kind regards

ei8ht

**Comment by oliverw** [comment]
What's a dedicated payout?

**Reply by ei8ht187** [reply]
A node that is just doing payouts. Not the master, not a stratum. It is an isolated node, not directly accessible from the internet. This is also the node that has the private key in our setup for all the coins. 
But in the meantime I figured it out, completly forgot about this thread. 

Best is to wait until your ERGO Fullnode with private keys is fully in sync. Then check the swagger API for the miningPubKey and add this to your ergo daemons config with the miningPubKeyHex option for the public reachable full nodes.
